src/test/java/pages/AllSongPage.java

```diff
@@ -0,0 +1,103 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class AllSongPage extends BasePage {
+
+    @FindBy(css = "section#songsWrapper tr:nth-child(1) td.title")
+    //".all-songs tr.song-item:nth-child(1)"
+    WebElement firstSong;
+    @FindBy(css = "section.music a.songs")
+    WebElement allSongBtn;
+    @FindBy(xpath = "//a[contains(text(), 'All Songs')]")
+    WebElement allSongBtnXpath;
+    @FindBy(css = "//div[@data-testid='sound-bar-play']")
+    WebElement soundBar;
+    @FindBy(css = "#songsWrapper > header > div.song-list-controls > span > button.btn-add-to")
+    WebElement addToPlayList;
+    @FindBy(css = "#songsWrapper > header > div.song-list-controls > div > section.existing-playlists > ul > li.favorites")
+    WebElement favoritesPlayList;
+    @FindBy(xpath = "//i[@title='Create a new playlist']")
+    WebElement buttonPlus;
+    @FindBy(xpath = "//li[text()='New Playlist']")
+    WebElement newPlayListButton;
+    @FindBy(xpath = "//i[@data-testid='play-next-btn']")
+    WebElement playNextButton;
+    @FindBy(xpath = "//span[@data-testid='play-btn']")
+    WebElement playButton;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as **private.**


I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```